cmd/tools/db-seeder/main.go

```diff
@@ -0,0 +1,205 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"flag"
+	"fmt"
+	"path"
+	"sync"
+	"time"
+
+	"github.com/jackc/pgx/v5/pgxpool"
```

Comment:
The rest of sc-bos uses pgx v4, is there a reason this can't also use the same version?